presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -654,6 +654,83 @@ This query will collect statistics for 2 partitions with keys:
 * ``partition1_value1, partition1_value2``
 * ``partition2_value1, partition2_value2``
 
+Quick Stats
+--------------------------------------
+
+The Hive connector can build basic statistics for partitions with missing statistics
+by examining file or table metadata. For example, parquet footers can be used to infer
+row counts, number of null and min/max values. These statistics help in query planning,
+and serve as interim cache of stats for partitions which haven't yet been ANALYZEd
```

Comment:
```suggestion
and serve as an interim cache of stats for partitions which haven't yet been ANALYZEd. 
```
Minor grammar and punctuation. ANALYZEd feels a little odd - not wrong, but I'm not sure when I read this what it means. Do you have another way that you could describe what you mean here?

Reply:
I've rewritten this to `These 'quick' statistics help in query planning,
and serve as as a temporary source of stats for partitions which haven't had ANALYZE run on
them.`